integration-tests/suites/common/collector_manager.go

```diff
@@ -138,25 +168,32 @@ func (c *CollectorManager) launchCollector() error {
 		"--privileged",
 		"--network=host"}
 
-	if !c.BootstrapOnly {
+	if !c.bootstrapOnly {
 		cmd = append(cmd, "-d")
 	}
 
-	for dst, src := range c.Mounts {
+	for dst, src := range c.mounts {
 		mount := src + ":" + dst
 		if src == "" {
 			// allows specification of anonymous volumes
 			mount = dst
 		}
 		cmd = append(cmd, "-v", mount)
 	}
-	for k, v := range c.Env {
+
+	for k, v := range c.env {
 		cmd = append(cmd, "--env", k+"="+v)
 	}
 
-	cmd = append(cmd, c.CollectorImage)
+	configJson, err := json.Marshal(c.config)
+	if err != nil {
+		return err
+	}
```

Comment:
I'd pay for go to implement something like the `?` from rust so we could do `configJson := json.Marshal(c.config)?` and be done...